local-volume/README.md

```diff
@@ -165,30 +165,39 @@ $ kubectl create -f bootstrapper/deployment/kubernetes/example-storageclass.yaml
 
 ### Step 3: Creating local persistent volumes
 
-#### Option 1: Bootstrapping the external static provisioner
+#### Option 1: Using the local volume static provisioner 
 
-This is optional, only for automated creation and cleanup of local volumes. See
-[bootstrapper/](./bootstrapper) and [provisioner/](./provisioner) for details and
-sample configuration files.
+	1. Install helm
+See README.md located in ./local-volume/helm directory.
 
-1. Create an admin account with cluster admin priviledge:
+	2. Create an admin account with cluster admin priviledge:
 ``` console
-$ kubectl create -f bootstrapper/deployment/kubernetes/<version>/admin-account.yaml
+$ kubectl create -f local-volume/provisioner/deployment/kubernetes/admin_account.yaml  
 ```
+	3. Generate Provisioner and config map spec, and customeze it.
+In the simplest form, to generate Provisioner spec, you run:
 
-2. Create a ConfigMap with your local storage configuration details.
 ``` console
-$ kubectl create -f bootstrapper/deployment/kubernetes/<version>/example-config.yaml
+helm template local-volume/helm/provisioner > provisioner.yaml
```

Comment:
I think provisioner.yaml should land inside local-volume/provisioner/deployment/kubernetes. For example, `local-volume/provisioner/deployment/kubernetes/provisioner.yaml` as the output location for the helm provisioner command. 

Nit: I think it might be worth mentioning that this file is generated, by naming it `provisioner_generated.yaml`

Reply:
done